js/script.js

```diff
@@ -19,7 +22,7 @@ if (unscroll) {
 if (!mute) {
   var citations = document.querySelectorAll('.citation');
   citations.forEach(function(citation, i){
-    citation.innerHTML = i+1;
+    citation.innerHTML = number_formatter.format(i+1);
```

Comment:
is there actually a situation where this number would be different depending on the locale? I think the highest number it goes to is around 30, so it's not large enough to have a decimal or comma.

Reply:
@MKorostoff 
No the point is not the decimal or comma.

Just because of different languages format in representing numbers I've added the formatter.
For example Japanese, Arabic, Persian, English and etc. have different numbers representations and characters.
For Example in English we see the citation number in this format:

![image](https://user-images.githubusercontent.com/45296858/147360829-25082e39-bd30-4da2-88b5-dae6f08755b9.png)

But for Persian I don't want to see English numbers, So I passed "fa-IR" parameter to the constructor and now I see citation numbers in this format:
![image](https://user-images.githubusercontent.com/45296858/147360777-7332d3e5-b46a-4305-896b-6c6ac3d45c52.png)

Just because of importance of language number representation I've added this.

Reply:
@SMSadegh19 I see, makes sense. I'll have to test it a little, but in principle this makes sense and I'll be happy to merge it after testing.

Reply:
Hi Matt! @MKorostoff 
Did you test it?